<?php
include('header.php');
?>

<!-- ####################################################################################################### -->
<div class="wrapper row3">
  <div class="rnd">
    <div id="container" class="clear"> 
      <!-- ####################################################################################################### -->
	  
      <h1>Epigenetic Data Browsers and Repositories</h1>
	  
	  <p>Go ahead, get your feet wet:</p>
	  <li><a href="http://epigenomesportal.ca/ihec/">IHEC Data Portal</a>: The International Human Epigenome Consortium (IHEC) brings forth reference epigenomes relevant to health and disease. View, search, and download all the data.</li>
	  <li><a href="http://egg2.wustl.edu/roadmap/web_portal/">ROADMAP Epigenomics</a>: The NIH Roadmap Epigenomics Mapping Consortium offers maps of histone modifications, chromatin accessibility, DNA methylation, and mRNA expression across 100s of human cell types and tissues.</li>
	  <li><a href="http://www.epigenomes.ca/data-release/">CEEHRC Platform</a>: A reference epigenome project for human cells and not the typical stem cell lines.</li>
	  <li><a href="http://deepblue.mpi-inf.mpg.de">DeepBlue</a>: Store and work with genomic and epigenomic data from a number of international consortiums.</li>
	  <li><a href="http://www.epigenomebrowser.org/" target="_blank" rel="noopener noreferrer">Epigenome Browser</a>: For the UCSC genome browser fans.</li>
	  <li><a href="http://epigenomegateway.wustl.edu/browser/" target="_blank" rel="noopener noreferrer">WashU Epigenome Browser</a>: A web browser that fffers tracks from ENCODE and Roadmap Epigenomics projects.</li>
	  <li><a href="http://useast.ensembl.org/info/website/tutorials/encode.html">Ensembl</a>: Featuring ENCODE.</li>
	  <li><a href="http://gralggen.lsi.upc.edu/recerca/genexp/" target="_blank" rel="noopener noreferrer">GenExp</a>: A web-based visualization tool to interactively explore a genomic database.</li>
	  <li><a href="http://www.genboree.org/epigenomeatlas/index.rhtml" target="_blank" rel="noopener noreferrer">The Epigenome Atlas</a>: Human reference epigenomes.</li>
	  <li><a href="http://www.edgar-wingender.de/huTF_classification.html">Classification of Human Transcription Factors</a>: The mother list of transcription factors and their binding sites.</li>
	  <p></p>
	  
      <h2>Epigenetic Databases</h2>
	  <p>While you're a pioneer, you're certainly not the first one to tread these waters. Go have a look at how lifetime's of work:</p>
	  
	  <p>Chromatin</p>
	  <li><a href="http://4dgenome.research.chop.edu">4DGenome</a>: A database of chromatin interactions across five species. Includes data from 3C, 4C, 5C, ChIA-PET, Hi-C, Capture-C, and IM-PET.</li>
      <li><a href="http://3cdb.big.ac.cn">3CDB</a>: A manually curated chromosome conformation capture (3C) database.</li>
      <li><a href="http://www.actrec.gov.in/histome/index.php" target="_blank" rel="noopener noreferrer">Histome</a>: A database of human histone variants, sites of post-translational modifications, and histone modifying enzymes.</li>
      <li><a href="http://www.cisred.org/" target="_blank" rel="noopener noreferrer">cisRED</a>: A motif database.</li>
      <li><a href="http://www.regulomedb.org/" target="_blank" rel="noopener noreferrer">RegulomeDB</a>: Annotation of functional variation in personal genomes using RegulomeDB.</li>
	  <p>DNA Methylation:</p>
	  <li><a href="http://www.neuroepigenomics.org/methylomedb/" target="_blank" rel="noopener noreferrer">MethylomeDB</a>: The Brain Methylome Database provides DNA methylation profiles from humans and mice.</li>
      <li><a href="http://202.97.205.78/diseasemeth/" target="_blank" rel="noopener noreferrer">DiseaseMeth</a>: Methylomes of human disease.</li>
      <li><a href="http://bioinfo2.ugr.es/NGSmethDB/" target="_blank" rel="noopener noreferrer">NGSmethDB</a>: Whole-genome bisulfite sequencing (WGBS) database for many different tissues, pathological conditions, and species.</li>
      <li><a href="http://smithlab.usc.edu/methbase/" target="_blank" rel="noopener noreferrer">MethBase</a>: Hundreds of methylomes from well studied organisms.</li>

	  <p>Noncoding RNA:</p>
	  <li><a href="http://bioinfo.life.hust.edu.cn/lncRNASNP/">lncRNASNP</a>: a database of SNPs in lncRNAs and their potential functions in human and mouse.</li>
      <li><a href="http://zmf.umm.uni-heidelberg.de/apps/zmf/mirwalk2/">miRWalk 2.0</a>: miRNA binding sites within the complete sequence of a gene and a comparison of binding sites from 12 existing miRNA-target prediction programs.</li>
      <li><a href="https://cm.jefferson.edu/rna22v2/">RNA22v2</a>: Get your miRNA targeting on with an unbiased algorithim that not only considers 3&#8217;UTR binding but also 5&#8217;UTR binding.</li>
      <li><a href="http://www.mirbase.org/" target="_blank" rel="noopener noreferrer">miRBase</a>: Published miRNA sequences.</li>
      <li><a href="http://diana.imis.athena-innovation.gr/DianaTools/index.php">DIANA Tools</a>: A suite of tools that include target prediction algorithms and experimentally verified miRNA targets.</li>
      <li><a href="http://www.targetscan.org/">TargetScan</a>: Predicts miRNA targets by searching for conserved sites that match the seed of each miRNA.  There are different version for humans, mouse, worm, fly, and fish.</li>
      <li><a href="http://www.ebi.ac.uk/enright-srv/microcosm/htdocs/targets/v5/">MicroCosm Targets</a>: Predicted targets for microRNAs across many species.</li>
      <li><a href="http://lemur.amu.edu.pl/share/php/mirnest/" target="_blank" rel="noopener noreferrer">miRNEST 2.0</a>: A database of animal, plant, and virus microRNAs.</li>
      <li><a href="http://www.noncode.org/" target="_blank" rel="noopener noreferrer">NonCode</a>: A database of all kinds of noncoding RNAs (except tRNAs and rRNAs) for 16 species.</li>
      <li><a href="http://www.broadinstitute.org/genome_bio/human_lincrnas/" target="_blank" rel="noopener noreferrer">Human lincRNA Catalog</a>:  A detailed human lincRNA catalogue based on ~4 billion RNA-Seq reads across 24 tissues and cell types.</li>
      <li><a href="http://compbio.uthsc.edu/miRSNP/" target="_blank" rel="noopener noreferrer">PolymiRTS</a>: Linking sequence to trait, check out what polymorphisms in your microRNA can do.</li>
      <li><a href="http://circnet.mbc.nctu.edu.tw">CircNET</a>: Offers profiles of tissue-specific circular RNA (circRNA) expression as well as circRNA-miRNA-gene regulatory networks.</li>
      <li><a href="http://circbase.org">circBase</a>: Public circRNA data and custom python scripts for circRNA discovery in your own (ribominus) RNA-seq data.</li>
      <li><a href="http://starbase.sysu.edu.cn/index.php">starBase v2.0</a>: Decode interaction networks of lncRNAs, miRNAs, circRNAs, RNA-binding proteins (RBPs), and mRNAs from large-scale CLIP-Seq data.</li>
      <li><a href="http://gyanxet-beta.com/circdb/">Circ2Traits</a>: Associate your circRNAs with diseases or traits.</li>

	  <p>Post-transcriptional regulation:</p>
	  <li><a href="http://www.rbp-var.biols.ac.cn/">RBP-Var</a>: a database of functional variants involved in regulation mediated by RNA-binding proteins.</li>
      <li><a href="http://POSTAR.ncrnalab.org">POSTAR</a>: a platform for exploring post-transcriptional regulation coordinated by RNA-binding proteins.</li>
      <li><a href="http://lulab.life.tsinghua.edu.cn/clipdb/">CLIPdb</a>: a CLIP-seq database for protein-RNA interactions.</li>
	  <p></p>
	  
      <h3>Epigenetic Tools for Statistical Data Analysis and Visualization</h3>
	  <p>Results are great! Now go do something with them. Use these data analysis and visualization tools to help decipher your data. Need to get a little intro into biostatistics? Go <a href="http://swirlstats.com/students.html">learn some R </a>and check out the <a href="http://www.bioconductor.org/">essentials from the bioconductor database</a> to get you started with data that won&#8217;t analyze the easy way. Here are a few of the best data analysis and visualization packages out there:</p>
	  
	  <p>DNA Methylation:</p>
	  <li><a href="http://bioconductor.org/packages/release/bioc/html/bsseq.html">Bsseq</a>: A Bioconductor (R) package that offers a suite of tools for analyzing and visualizing your very own WGBS data towards your goal of identifying differentially methylated regions (DMRs).</li>
      <li><a href="https://github.com/thomasvangurp/epiGbs">epiGbs</a>: A reference genome free reduced representation bisulfite sequencing (RRBS) method that enables cost-effective analysis of DNA methylation and genetic variation in hundreds of samples.</li>
      <li><a href="http://smithlab.usc.edu/methpipe/">MethPipe</a>: Analyzes your WGBS, and RRBS data to identify DMRs, allele-specific methylation, and partially methylated domains.</li>
      <li><a href="https://bioconductor.org/packages/release/bioc/html/M3D.html" target="_blank" rel="noopener noreferrer">M3D:</a> A Bioconductor (R) package that uses a kernel methods to identify DMRs.</li>
      <li><a href="https://code.google.com/p/moabs/">MOABS</a>: Bioinformatic method for aligning your WGBS data and detecting DMRs.</li>
      <li><a href="http://biochem.otago.ac.nz/research/databases-software/">DMAP</a>: A (C-based) tool for RRBS and WGBS data, which includes a suite of statistical tools and a different investigating approach for analysing DNA methylation data and it also links any list of regions to the genome and provides gene and CpG features.  It now features a novel fragment based analysis for RRBS, which has not been shown before.</li>
      <li><a href="http://www.bioconductor.org/packages/release/bioc/html/BEAT.html">BEAT</a>: A Bioconductor (R) package that lets you analyze single-cell BS-seq data.</li>
      <li><a href="http://bioconductor.org/packages/release/bioc/html/methylPipe.html">methylPipe</a>: A Bioconductor (R) package for the analysis of CpG and non-CpG methylation from WGBS data that also enables integration with other epigenomic data sets.</li>
      <li><a href="http://www.bioconductor.org/packages/release/bioc/html/compEpiTools.html">compEpiTools</a>: A Bioconductor (R) package that helps you analyze, integrate, and visualize multiple epigenomic data sets.</li>
      <li><a href="https://www.bioconductor.org/packages/release/bioc/html/DMRcate.html" target="_blank" rel="noopener noreferrer">DMRcate</a>: A Bioconductor (R) package for DMR identification from the human genome using WGBS and Illumina Infinium array (450K and EPIC) data.</li>
	  <li><a href="http://www.bioconductor.org/packages/release/bioc/html/minfi.html">Minfi</a>: A Bioconductor (R) package for your Illumina Infinium arrays (450K and EPIC) that provides comprehensive analysis and takes cellular heterogeneity into account, after all variety is the spice to life.</li>
	  <li><a href="http://bioconductor.org/packages/release/bioc/html/ChAMP.html">ChAMP</a>: A Bioconductor (R) package that offers QC/QA metrics and a number of normalization methods in order to identify DMRs and copy number variations in Illumina Infinium array (450K and EPIC) data.</li>
	  <li><a href="http://www.bioconductor.org/packages/release/bioc/html/FEM.html">FEM</a>: A Bioconductor (R) package that offers integrative analysis of DNA methylation and gene expression data.</li>
	  <li><a href="http://www.bioconductor.org/packages/release/bioc/html/coMET.html">coMET</a>: A Bioconductor (R) package for the visualisation of Epigenome-Wide Association Study (EWAS) from a genomic region perspective.</li>
	  <li><a href="http://www.bioconductor.org/packages/release/bioc/html/Repitools.html">Repitools</a>: A Bioconductor (R) package for the analysis of enrichment-based DNA methylation data.</li>
	  <li><a href="http://rnbeads.mpi-inf.mpg.de">RnBeads</a>: an R package for comprehensive analysis of DNA methylation data from Illumina Infinium arrays (450K and EPIC) and BS-seq. MeDIP-seq and MBD-seq are also supported after some external processing.</li>
	  <li><a href="https://bioconductor.org/packages/release/bioc/html/SMITE.html">SMITE</a>: A Bioconductor (R) package for Significance-based Modules Integrating the Transcriptome and Epigenome.</li>
	  <li><a href="http://journal.frontiersin.org/Journal/10.3389/fgene.2014.00325/abstract">DaVIE</a>: An intuitive user interface to perform visual comparisons across all your large DNA methylation data sets.</li>

	  <p>ChIP-seq:</p>
	  <li><a href="http://liulab.dfci.harvard.edu/MACS/">MACS</a>: Model-based Analysis of ChIP-seq (MACS) is a go to peak-finding algorithm.</li>
      <li><a href="http://home.gwu.edu/~wpeng/Software.htm">SICER</a>: A clustering approach for identification of enriched domains from histone modification ChIP-Seq data.</li>
	  <li><a href="https://manticore.niehs.nih.gov/pavis2">PAVIS</a>: PAVIS (Peak Annotation and Visualization) lets you annotate and visualize your ChIP-seq and BS-seq data.</li>
	  <li><a href="http://easeq.net">EaSeq</a>: Lets you analyze and visualize your ChIP-seq data with graphical user interface that runs on a typical PC.</li>
	  <li><a href="http://costalab.org/wp/odin">ODIN</a>: A ChIP-seq tool that not only detects peaks but can also call and provide statistics on differential peaks between two conditions.</li>
	  <li><a href="http://www.bioconductor.org/packages/3.2/bioc/html/MMDiff.html" target="_blank" rel="noopener noreferrer">MMDiff</a>: A Bioconductor (R) package that detect differential peaks in your ChIP-seq data.</li>
	  <li><a href="http://www.bcgsc.ca/platform/bioinfo/software/alea">ALEA</a>: Lets you analyze ChIP-seq or RNA-seq data to correlate allele-specific differences with epigenomic status.</li>
	  <li><a href="http://biogpu.ddns.comp.nus.edu.sg/~chipseq/webseqtools2/TASKS/Motif_Enrichment/submit.php?email=guest" target="_blank" rel="noopener noreferrer">CENTDIST</a>: A web-application that identifies transcription factors hanging around your ChIP-seq peaks.</li>
	  <li><a href="http://jjwanglab.org/chip-array-v2/about" target="_blank" rel="noopener noreferrer">ChIP-Array v2.0</a>: Integrate your ChIP-seq or ChIP-CHIP data with gene expression to build a regulatory network. Works for human, mouse, yeast, fly, and arabidopsis data.</li>
	  <li><a href="http://tanlab4generegulation.org/CoSBIWebpage.html">CosBI</a>: Combinatorial chromatin modification patterns in the human genome revealed by subspace clustering.</li>
	  <li><a href="http://159.149.160.51/pscan_chip_dev/" target="_blank" rel="noopener noreferrer">Pscan-ChIP</a>: A web server that scans ChIP-seq peak coordinates for over-representated transcription factor binding site motifs.</li>
	  <li><a href="http://www.bioconductor.org/packages/release/bioc/html/chroGPS.html">chroGPS</a>: A Bioconductor (R) package aimed at integration, visualization, and functional analysis of epigenomics data.</li>
	  <li><a href="http://www.bioconductor.org/packages/release/bioc/html/epigenomix.html">Epigenomix</a>: A Bioconductor (R) package that lets you integrate your RNA-seq or microarray data with your ChIP-seq data. It lets you preprocess and create differential gene lists for both data sets.</li>
	  <li><a href="http://compbio.mit.edu/ChromHMM/">ChromHMM</a>: automating chromatin-state discovery and characterization.</li>
	  <li><a href="http://noble.gs.washington.edu/proj/segway/">Segway</a>:Unsupervised pattern discovery in human chromatin structure through genomic segmentation.</li>

	  <p>Motif Discovery:</p>
	  <li><a href="http://wanglab.ucsd.edu/star/epigram/">Epigram:</a> An analysis pipeline that predicts histone modification and DNA methylation patterns from DNA motifs. <a href="http://epigenie.com/epigram-predicting-the-human-epigenome-from-dna-motifs/">Check out our coverage</a>.</li>
	  <li><a href="http://homer.salk.edu/homer/motif/">Homer</a>: Discover motifs critical to the differences between your sample groups. The art is just a bonus.</li>
	  <li><a href="http://meme-suite.org" target="_blank" rel="noopener noreferrer">The MEME Suite:</a> Motif Based Sequence Analysis Tools.</li>
	  <li><a href="http://dire.dcode.org/" target="_blank" rel="noopener noreferrer">DiRE</a>: A web server for predicting distant (outside of proximal promoter regions) regulatory elements (DiRE) of co-regulated genes.</li>
	  <li><a href="http://melina2.hgc.jp/public/index.html" target="_blank" rel="noopener noreferrer">Melina</a>: (Motif Elucidator in Nucleotide Sequence Assembly) can run multiple motif prediction tools simultaneously.</li>

	  <p>RNA:</p>
	  <li><a href="https://biowardrobe.com">BioWardrobe</a>: Lets you store, visualize, analyze and integrate epigenomic and transcriptomic data using a web-based graphical user interface that doesn&#8217;t require programming expertize.</li>
	  <li><a href="http://galaxyproject.org/" target="_blank" rel="noopener noreferrer">Galaxy</a>: Provides an interface to help you with all the fancy code needed for RNA-seq.</li>
	  <li><a href="http://www.babelomics.org">Babelomics 5</a>: A user-friendly interface for a suite of tools for gene expression and genomic data.</li>
	  <li><a href="http://www.htslib.org">Samtools</a>: A suite of programs for interacting with high-throughput sequencing data and formats that include SAM/BAM/CRAM as well as BCF2/VCF/gVCF.</li>
	  <li><a href="https://broadinstitute.github.io/picard/">Picard</a>: A set of command line tools for file types such as SAM/BAM/CRAM and VCF.</li>
	  <li><a href="https://github.com/dstreett/Super-Deduper">Super-Deduper</a>: Remove PCR duplicates from your paired-end reads.</li>
	  <li><a href="https://github.com/dstreett/FLASH2">FLASH 2</a>: Merge paired-end reads.</li>
	  <li><a href="https://github.com/ucdavis-bioinformatics/scythe">Scythe</a>: Remove sequencing adapters from your single-end reads.</li>
	  <li><a href="https://github.com/dstreett/sickle">Sickle</a>: Trim low quality regions from your RNA-seq reads.</li>
	  <li><a href="http://bowtie-bio.sourceforge.net/bowtie2/index.shtml">Bowtie 2</a>: Align your sequencing reads to long reference sequences.</li>
	  <li><a href="https://ccb.jhu.edu/software/tophat/index.shtml">Tophat</a>: A splice junction mapper for RNA-seq reads that uses Bowtie.</li>
	  <li><a href="https://github.com/alexdobin/STAR/releases">STAR</a>: RNA-seq aligner that peforms simultaneous read mapping and counting.</li>
	  <li><a href="http://cole-trapnell-lab.github.io/cufflinks/">Cufflinks</a>: Assembles transcripts, estimates their abundances, and tests for differential expression in RNA-Seq dats. It accepts aligned RNA-seq reads.</li>
	  <li><a href="http://compbio.mit.edu/cummeRbund/">CummeRbund</a>: A R package that helps with analyzing Cufflinks RNA-Seq output.</li>
	  <li><a href="https://bioconductor.org/packages/release/bioc/html/DESeq2.html">DESeq2</a>: Detect differential expression of transcripts in your RNA-seq data.</li>
	  <li><a href="https://bioconductor.org/packages/release/bioc/html/edgeR.html">EdgeR</a>: A Bioconductor (R) package for differential expression analysis of RNA-seq data.</li>
	  <li><a href="https://pachterlab.github.io/kallisto/about">Kallisto</a>: A program for quantifying abundances of transcripts from RNA-seq data. Uses pseudoalignment to skip the alignment step.</li>
	  <li><a href="https://combine-lab.github.io/salmon/">Salmon</a>: A tool for quantifying the expression of transcripts using RNA-seq data.</li>
	  <li><a href="http://www.bioconductor.org/packages/release/bioc/html/gcrma.html">GCRMA</a>: Pre-processing algorithm for affymetrix arrays.</li>

	  <p>Other Useful Tools:</p>
	  <li><a href="http://przemol.github.io/seqplots/">SeqPlots</a>: Exploratory data analysis and visualization tool bundled in a nice app that lets you generate some pretty pictures of your functional genomic features.</li>
	  <li><a href="http://deeptools.readthedocs.io/en/latest/index.html">deeptools</a>: tools for exploring deep sequencing data</li>
	  <li><a href="https://github.com/shenlab-sinai/ngsplot">ngs.plot</a>: Visualize your results at functional genomic regions.</li>
	  <li><a href="http://gat.readthedocs.io/en/latest/contents.html">GAT</a>: Genomic Association Tester (GAT) lets you compute the significance of the overlap between all your fancy data sets.</li>
	  <li><a href="https://bioconductor.org/packages/release/bioc/html/GeneOverlap.html">GeneOverlap</a>: A Bioconductor (R) package to statistically test and then visualize gene overlaps between multiple gene lists.</li>
	  <li><a href="http://fantom.gsc.riken.jp/zenbu/">ZENBU</a>: Japanese for all, entire, whole, altogether. This browser lets you integrate and interact with your data in a nice visual environment.</li>
	  <li><a href="http://epiexplorer.mpi-inf.mpg.de/" target="_blank" rel="noopener noreferrer">EpiExplorer</a>: Import your very own data and compare it to ENCODE.</li>
	  <li><a href="http://www.bionut.ki.se/users/pesv/podbat/index.html" onclick="__gaTracker('send', 'event', 'outbound-article', 'http://www.bionut.ki.se/users/pesv/podbat/index.html', 'Podbat');" title="Podbat"  target="_blank" rel="noopener noreferrer">Podbat</a>: A positioning database and analysis tool that takes data from a number of sources and to allow for the detailed dissection of a range of chromatin modifications. It can be used to analyze, visualize, store and share your data. Check out <a href="http://epigenie.com/podbat-lets-you-fly-through-epigenomic-analysis/" target="_blank" rel="noopener noreferrer">our coverage on Podbat</a>.</li>
	  <li><a href="http://insulatordb.uthsc.edu/" target="_blank" rel="noopener noreferrer">CTCF Insulator Database</a>: <em>In silico</em> prediction for all your genomic insulation needs!</li>
	  <li><a href="http://www.rsat.eu" target="_blank" rel="noopener noreferrer">Regulatory Sequence Analysis Tools</a>: Detects regulatory signals in non-coding sequences.</li>
	  <li><a href="http://zlab.bu.edu/CarrieServer/html/" target="_blank" rel="noopener noreferrer">CARRIE</a>: It takes takes your two-condition microarray data and applies a promoter analysis to infer a regulatory network.</li>
	  <p></p>

      <h4>Gene Ontology and Pathway Analysis</h4>
	  <p>So, you've got that wonderful omic data down to a nice little gene list. Now it's time to have fun figuring out just what they're all up to:</p>
	  <li><a href="http://amp.pharm.mssm.edu/Enrichr/">Enrichr</a>: Find out about transcriptional regulation, pathways, onotologies, and much more from this neat little web tool.</li>
	  <li><a href="http://cpdb.molgen.mpg.de/CPDB">ConsensusPathwayDB</a>: A master tool that pulls from a large number of databases to provide ontology and pathway analysis for humans, mice, and yeast.</li>
	  <li><a href="http://bejerano.stanford.edu/great/public/html/">GREAT</a>: Genomic Regions Enrichment of Annotations Tool (GREAT) gives biological context to non-coding genomic regions by analyzing the annotations of the nearby genes. It&#8217;s great for analyzing genomic coordinates from your ChIP-seq and DNA methylation data.</li>
	  <li><a href="http://labs.genetics.ucla.edu/horvath/CoexpressionNetwork/Rpackages/WGCNA/#abst">WGCNA</a>: an R package for weighted correlation network analysis that can be used to find correlated gene clusters.</li>
	  <li><a href="http://www.broadinstitute.org/gsea/index.jsp">Gene Set Enrichment Analysis (GSEA)</a>: The name says it all, this pioneering program lets you compare against the Molecular Signatures Database (MSigDB).</li>
	  <li><a href="https://david.ncifcrf.gov">The Database for Annotation, Visualization and Integrated Discovery (DAVID)</a>: The granddaddy of them all.</li>
	  <li><a href="http://www.webgestalt.org/option.php">WebGestalt</a>: a more comprehensive, powerful, flexible and interactive gene set enrichment analysis toolkit. </li>
	  <li><a href="http://string-db.org">STRING</a>: STRING is a database of known and predicted protein interactions that lets you visualzie interacting networks.</li>
	  <li><a href="http://www.genemania.org">GeneMANIA</a>: GeneMANIA lets you visualize your gene lists and finds other related genes by using a very large set of functional association data.</li>
	  <li><a href="http://www.genmapp.org/go_elite/">GO Elite</a>: Ontology and pathway analysis from multiple databases.</li>
	  <li><a href="http://opossum.cisreg.ca/oPOSSUM3/" target="_blank" rel="noopener noreferrer">oPOSSUM</a>: Analyze your gene list for transcription factor binding sites.</li>
	  <li><a href="http://revigo.irb.hr">REVIGO</a>: Shorten your long list of Gene Ontology terms by removing redundant ones.</li>
	  <li><a href="http://biit.cs.ut.ee/gprofiler/index.cgi">g:Profiler</a>: Ontologies, pathways, and more from your gene list. Currently available for 200+ species.</li>
	  <li><a href="https://toppgene.cchmc.org">ToppGene</a>: A suite of tools to see what is enriched for in your gene list.</li>
	  <li><a href="https://endeavour.esat.kuleuven.be/">Endeavour</a>: A web resource for gene prioritization in multiple species.</li>
	  <p></p>

      <h5>Sodium Bisulfite Primer Design</h5>
	  <p>When Bisulfite reduces the complexity of a sequence, it increases the complexity of it's primer design. These programs help take the pain out of bisulfite primer design:</p>
	  <li><a href="http://bisearch.enzim.hu/" target="_blank" rel="noopener noreferrer">BiSearch</a>: A primer-design algorithm that can be with both bisulfite converted and non-converted sequences.</li>
	  <li><a href="http://www.urogene.org/methprimer" target="_blank" rel="noopener noreferrer">MethPrimer</a>: A program that lets you design primers for bisulfite PCR that also predicts CpG islands in DNA sequences.  It lets you design primers for Methylation-Specific PCR (MSP), Bisulfite-Sequencing PCR (BSP), and Bisulfite-Restriction PCR.</li>
	  <li><a href="http://www.zymoresearch.com/tools/bisulfite-primer-seeker">Bisulfite Primer Seeker</a>: Zymo Research&#8217;s handy online bisulfite primer design tool, designed by experts.</li>
      <!-- ####################################################################################################### --> 
    </div>
  </div>
</div>

<?php
include('footer.php');
?>

